CompuCell3D/core/CompuCell3D/Potts3D/Potts3D.cpp

```diff
@@ -1381,12 +1381,14 @@ void Potts3D::update(CC3DXMLElement *_xmlData, bool _fullInitFlag) {
 
             //we do not allow steering for motility specified by type

```

Comment:
Since steering is not allowed, it's odd that this code is in `update()`. However, I would avoid a refactor.

Reply:
Unsure whether this rule still applies. @maciekswat ?